bftengine/src/bftengine/ReplicasInfo.cpp

```diff
@@ -63,6 +63,11 @@ ReplicasInfo::ReplicasInfo(const ReplicaConfig& config,
       _dynamicCollectorForPartialProofs{dynamicCollectorForPartialProofs},
       _dynamicCollectorForExecutionProofs{dynamicCollectorForExecutionProofs},
 
+      /*
+       * Ids order is: [replicas, ro-replicas, clientProxies, client-service external clients, client-service ID,
+       * operator, internal-clients]. Notice, that in case we have an operator, it is included in numOfExternalClients,
```

Comment:
-> client-service clients (aka external clients)
Please mention also cre client. 
BTW, there is a configuration parameter for cre presence, should we take it into account or remove it if not relevant anymore?

Reply:
the cre configuration parameter is only for the client not for replica (because the client needs to know not to use the last client)